tests/core/test_reforms.py

```diff
@@ -43,6 +52,22 @@ def test_formula_neutralization():
     assert_near(disposable_income_reform, 0)
 
 
+def test_default_variable_neutralization():
```

Comment:
I don't understand what is being tested here 😕 
--> I feel `test_neutralized_variable_returns_default_value_when_computed` would be more explicit :)